Quantum-Control-Applications/Superconducting/Single Fixed Transmon/DRAG_calibration.py

```diff
@@ -89,48 +90,31 @@
     qm = qmm.open_qm(config)
 
     job = qm.execute(drag)
-    res_handles = job.result_handles
-    I1_handle = res_handles.get("I1")
-    Q1_handle = res_handles.get("Q1")
-    I2_handle = res_handles.get("I2")
-    Q2_handle = res_handles.get("Q2")
-    iteration_handle = res_handles.get("iteration")
-    I1_handle.wait_for_values(1)
-    Q1_handle.wait_for_values(1)
-    I2_handle.wait_for_values(1)
-    Q2_handle.wait_for_values(1)
-    iteration_handle.wait_for_values(1)
-    state_handle = res_handles.get("state1")
-    state_handle.wait_for_values(1)
-    state2_handle = res_handles.get("state2")
-    state2_handle.wait_for_values(1)
-    next_percent = 0.1  # First time print 10%
-
-    def on_close(event):
-        event.canvas.stop_event_loop()
-        job.halt()
-
-    f = plt.figure()
-    f.canvas.mpl_connect("close_event", on_close)
-    print("Progress =", end=" ")
-
-    while res_handles.is_processing():
-        iteration = iteration_handle.fetch_all()
-        if iteration / n_avg > next_percent:
-            percent = 10 * round(iteration / n_avg * 10)  # Round to nearest 10%
-            print(f"{percent}%", end=" ")
-            next_percent = percent / 100 + 0.1  # Print every 10%
+    # Get results from QUA program
+    results = fetching_tool(job, data_list=["I1", "I2", "Q1", "Q2", "state1", "state2", "iteration"], mode="live")
```

Comment:
Yes, this is the only reason :)
There are two ways: (1) either I remove the unused variables from data_list so that we don't fetch them at all and i could actually also remove the save statements from the qua programs, (2) or keep it as is if it can be useful for debugging purposes, I don't really have an opinion since I never used the script so as you want :) Maybe @KevinAVR has a preference

Reply:
Well, not really.
I understand why we might want to keep it in the program (and stream processing).
But if we don't even save it to a variable in python, then there is no reason to put it into the fetching tool.
So either:
1. Remove it from the fetching tool
2. Fetch the variables from the fetching tool. Even if you don't use them, they'll be available in the python console/notebook
3. Remove them all together (Which Kevin already convinced us not to do).

Reply:
Alright I see your point, I just removed the python variable (replaced by _) in order to get rid of the pycharm warning saying that these variables are not used... My preference goes towards removing them from the fetching tool